packages/kbn-guided-onboarding/src/components/landing_page/use_case_card.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { ReactNode } from 'react';
+import { EuiCard, EuiText, EuiImage } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { GuideId } from '../../types';
+
+type UseCaseConstants = {
+  [key in UseCase]: {
+    logAltText: string;
+    betaBadgeLabel: string;
+  };
+};
+const constants: UseCaseConstants = {
+  search: {
+    logAltText: i18n.translate('guidedOnboardingPackage.gettingStarted.search.iconName', {
+      defaultMessage: 'Enterprise Search logo',
+    }),
+    betaBadgeLabel: i18n.translate('guidedOnboardingPackage.gettingStarted.search.betaBadgeLabel', {
+      defaultMessage: 'search',
+    }),
+  },
+  observability: {
+    logAltText: i18n.translate('guidedOnboardingPackage.gettingStarted.observability.iconName', {
+      defaultMessage: 'Observability logo',
+    }),
+    betaBadgeLabel: i18n.translate(
+      'guidedOnboardingPackage.gettingStarted.observability.betaBadgeLabel',
+      {
+        defaultMessage: 'observe',
+      }
+    ),
+  },
+  security: {
+    logAltText: i18n.translate('guidedOnboardingPackage.gettingStarted.security.iconName', {
+      defaultMessage: 'Security logo',
+    }),
+    betaBadgeLabel: i18n.translate(
+      'guidedOnboardingPackage.gettingStarted.security.betaBadgeLabel',
+      {
+        defaultMessage: 'security',
+      }
+    ),
+  },
+};
+
+export type UseCase = 'search' | 'observability' | 'security';
```

Comment:
I'm curious why this type is necessary, when it matches `GuideId`.  Appears to me these types need coalescing.